src/Hedgehog.Experimental/Hedgehog.Experimental.fsproj

```diff
@@ -35,7 +35,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Update="FSharp.Core" Version="4.3.4" />
+    <PackageReference Update="FSharp.Core" Version="4.6.2" />
```

Comment:
1. Because the test project already uses `4.6.2` so at least be consistent within the solution
2. Because `Seq.tryExactlyOne` was introduced years ago but still after even more ancient `4.3.4`

Reply:
These are not good enough reasons to change this.  It is fine for the test code to use the latest version of `FSharp.Core`.  That allows us to use new things like `Seq.tryExactlyOnce`.  Increasing the version of `FSharp.Core` places stricter requirements on users.

It is easy to implement your optimization while continuing to use version `4.3.4` of `FSharp.Core`.  I just forced pushed to the branch after switching to this code:

```fsharp
 |> List.map (snd >> function
                     | [p] -> p
                     | ps -> ps |> List.sortByDescending getDepth |> List.head)
```

Reply:
It is not latest BTW, it is at least 3 years old. Latest is `6.x.x`.